sources/packages/backend/libs/test-utils/src/factories/application.ts

```diff
@@ -19,6 +19,7 @@ export function createFakeApplication(relations?: {
   const application = new Application();
   application.data = {} as ApplicationData;
   application.programYear = relations?.programYear ?? createFakeProgramYear();
+  application.programYear.id = 2;
```

Comment:
Can you elaborate on why we need to have it hard-coded here always to use id 2?
Was it here causing some issues?

Reply:
yes, the previous createFakeProgramYear() used that id as the programYear is already in the DB.